docs/src/index.md

```diff
@@ -83,6 +83,8 @@ integrated they are with DataFrames.jl.
       plotting library that uses a different "grammar of graphics" syntax and has
       an emphasis on interactive graphics.
 - **Data Wrangling**:
+    - [Impute.jl](https://github.com/invenia/Impute.jl):
+      various methods for handling missing data in Vectors, Matrices and Tables.
```

Comment:
```suggestion
      various methods for handling missing data in vectors, matrices and tables.
```

Reply:
:) - I copy pasted the sentence from their web page